dts/bindings/clock/ite,it8xxx2-ecpm.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: ITE it8xxx2 ECPM (EC Clock and Power Management Controller) node
+
+compatible: "ite,it8xxx2-ecpm"
+
+include: [clock-controller.yaml, base.yaml]
+
+properties:
+    reg:
+      required: true
+
+    label:
```

Comment:
remove `label` property

Reply:
Got it, sir. The label property has been removed.

Reply:
@galak

Thanks again for guidance and revisiting.
If there's any further idea, please kindly let me know.